src/tests/Microsoft.PowerFx.Interpreter.Tests.Shared/LanguageServiceProtocol/LegacyLanguageServerTests.cs

```diff
@@ -2000,8 +1994,15 @@ public void ParseAndErrorLocaleAreDifferent()
             // Checking if contains text in the correct locale
             // the value should be localized. Resx files have this localized.
             // If it's a different error message, then we may have a bug in the parser locale. 
-            Assert.Contains("El nombre no es válido. No se reconoce \"foo\".", diags.First().Message);
-            Assert.Empty(exList);
+            Assert.Contains("El nombre no es válido. No se reconoce \"foo\".", diags.First().Message);
+            Assert.Empty(exList);
+
+            // Suggestions should provide localized descriptions, based on parser option locale.
+            var suggestions = _scopeFactory.GetOrCreateInstance("powerfx://app").Suggest("Value(", "Value(".Length);
+            var overload = suggestions.FunctionOverloads.First();
+
+            Assert.Equal("Convertit un « texte » représentant un nombre en valeur numérique.", overload.Definition);
```

Comment:
>"Convertit un « texte » représentant  [](http://example.com/codeflow?start=25&length=37)

We should probably be using the "error locale" , not parse locale.  Error locale is poorly named.